Seulement dans fvwm: borders.c.orig
Seulement dans fvwm: builtins.c.orig
Seulement dans fvwm: events.c.orig
Seulement dans fvwm: fvwm.h.orig
Seulement dans fvwm: menus.c.orig
Seulement dans fvwm: screen.h.orig
diff -U3 -r libs/defaults.h libs/defaults.h
--- libs/defaults.h	2011-08-15 18:25:40.000000000 +0200
+++ libs/defaults.h	2011-12-09 21:55:17.681582269 +0100
@@ -153,7 +153,7 @@
 #define BROKEN_MINSIZE_LIMIT 30000
 
 /* geometry window */
-#define GEOMETRY_WINDOW_BW                 2 /* pixels */
+#define GEOMETRY_WINDOW_BW                 1 /* pixels */
 #define GEOMETRY_WINDOW_STRING             " +8888 x +8888 "
 #define GEOMETRY_WINDOW_POS_STRING         " %+-4d %+-4d "
 #define GEOMETRY_WINDOW_SIZE_STRING        " %4d x %-4d "
diff -U3 -r modules/FvwmProxy/FvwmProxy.c modules/FvwmProxy/FvwmProxy.c
--- modules/FvwmProxy/FvwmProxy.c	2011-08-07 00:03:32.000000000 +0200
+++ modules/FvwmProxy/FvwmProxy.c	2011-12-09 21:55:17.681582269 +0100
@@ -1221,7 +1221,7 @@
 	{
 		XClearWindow(dpy,proxy->proxy);
 	}
-	RelieveRectangle(dpy,proxy->proxy, 0,0, w - 1,h - 1, hi_gc,sh_gc, 2);
+	RelieveRectangle(dpy,proxy->proxy, 0,0, w - 1,h - 1, hi_gc,sh_gc, 1);
 
 #if 0
 	if (proxy->iconname != NULL)
